lib/node_modules/@stdlib/math/base/special/factorial2/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env julia
```

Comment:
Definitely concur, needs some refactoring. I haven't actually done the investigation but I believe it's worth doing the comparison. Will do so and update you the results.